pkg/cloud/services/s3/s3_test.go

```diff
@@ -627,6 +627,8 @@ func TestDeleteObject(t *testing.T) {
 			},
 		}
 
+		s3Mock.EXPECT().HeadObject(gomock.Any()).Times(1)
```

Comment:
```suggestion
		s3Mock.EXPECT().HeadObject(gomock.Any())
```

since that's the same as `.Times(1)` [repeated below]